.github/workflows/clair-in-ci-db.yaml

```diff
@@ -37,15 +55,35 @@ jobs:
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
 
       - name: Pull-and-retag-image
+        if: ${{ github.event_name != 'pull_request' }}  # don't retag image from PR
         run: |
           docker pull ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }}
           docker tag ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.PREVIOUS_TAG }}
           docker push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.PREVIOUS_TAG }}
           docker rmi -f $(docker images -q)
 
-      - name: Build-and-push-image
-        id: build-and-push-image
+      - name: Build image
+        id: build-image
         run: |
           docker build . -t ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }} -f ./clair-in-ci/Dockerfile
+
+      - name: Get Clair version
+        run: |
+          docker run --rm -t ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }} clair-action --version
+
+      - name: Check Clair output format
+        run: |
+          # test real life usage of clair
+          mkdir results
+          docker run --rm -v $(pwd)/results:/results:rw -t ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }} bash -c 'clair-action report --image-ref=registry.access.redhat.com/ubi9-minimal --db-path=/tmp/matcher.db --format=quay > /results/clairdata.json'
+
+
+          # check if required metadata and path to values are the same as expected
+          jq -e '.data[].Features[0] | select(has("Name") and has("Vulnerabilities")) or error("Required keys does not exist")' results/clairdata.json
```

Comment:
tiniest of nits: `Required keys does not exist` -> `Required keys do not exist`

Reply:
fixed